### PR TITLE
Release 1.0.6

### PR DESCRIPTION
**Pull request by ddl-alexpanin**

### CODE REVIEW

**Comment by ddl-alexpanin** [comment]
@dknupp @ddl-gabrielhaim how does it look like?

File: README.md [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
 
 `python-domino` library is compatible with different versions of the `Domino`:
 
-| Domino Versions | Python-Domino |
-| --------------- |:-------------:|
-| 3.6.x or Lower  | [0.3.5](http://github.com/dominodatalab/python-domino/archive/0.3.5.zip)         |
-| 4.1.0 or Higher | [1.0.0](https://github.com/dominodatalab/python-domino/archive/1.0.0.zip) or Higher         |
+| Domino Versions |  Python-Domino  |
+| --------------- |:---------------:|
+| 3.6.x or Lower  | 0.3.5           |
```
Comment (dmitriyvolk) [inline README.md:13]: You removed links to the old version ZIPs, but the old versions are not on PyPy. Is that intended?

**Review by dmitriyvolk — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ddl-alexpanin) [reply README.md:13]: Well yes, old versions won't be in PyPI. Okay, links probably can go back.

**Comment by ddl-alexpanin** [comment]
Here's how it looks in test PyPI: https://test.pypi.org/project/dominodatalab-test/1.0.6/

**Comment by dknupp** [comment]
@ddl-alexpanin -- I will try to test the steps later today.

**Review by ddl-gabrielhaim — APPROVED** [review]
Looks good to me!

**Review by dknupp — CHANGES_REQUESTED** [review]
I have two immediate concerns.

1. Altering the top-level name of the package seems like a big deal. Is this a PM or customer requested change?
2. The install of the test package from test.pypi.org fails.
```
% pip install --index-url https://test.pypi.org/simple dominodatalab-test
Looking in indexes: https://test.pypi.org/simple, https://ddl-pypi-reader:****@domino.jfrog.io/domino/api/pypi/pypi-local/simple
Collecting dominodatalab-test
  Downloading https://test-files.pythonhosted.org/packages/88/10/d80f41c00bd6ae091698afe5236e6c64bbe2f932072e3c1847e03f08e66c/dominodatalab_test-1.0.6-py3-none-any.whl (27 kB)
Collecting requests>=2.4.2
  Downloading https://test-files.pythonhosted.org/packages/6d/00/8ed1b6ea43b10bfe28d08e6af29fd6aa5d8dab5e45ead9394a6268a2d2ec/requests-2.5.4.1-py2.py3-none-any.whl (468 kB)
     |████████████████████████████████| 468 kB 5.8 MB/s
ERROR: Could not find a version that satisfies the requirement bs4==0.*,>=0.0.1 (from dominodatalab-test)
ERROR: No matching distribution found for bs4==0.*,>=0.0.1
WARNING: You are using pip version 21.0.1; however, version 21.3.1 is available.
You should consider upgrading via the '/private/tmp/test_106/bin/python -m pip install --upgrade pip' command.
```

File: README.md [hunk 2]
```diff
@@ -1,37 +1,36 @@
...
+| --------------- |:---------------:|
+| 3.6.x or Lower  | [0.3.5](http://github.com/dominodatalab/python-domino/archive/0.3.5.zip)             |
+| 4.1.0 or Higher | [1.0.0](https://github.com/dominodatalab/python-domino/archive/1.0.0.zip) or Higher |
 
 ## Installation
 
-At this time, these Domino Python bindings are not in PyPi. You can install the latest version of this package from our Github `master` branch with the following:
+Starting from version `1.0.6`, `python-domino` is available on PyPI as `dominodatalab`:
+
+    pip install dominodatalab
```
[the remark this answers is not in the document]
Comment (ddl-alexpanin) [reply README.md:20]: 1. Shared a doc with a request to change top-level package name with you.
2. Use `pip install --extra-index-url  https://test.pypi.org/simple/ dominodatalab-test`. See https://stackoverflow.com/questions/60868060/module-on-test-pypi-cant-install-dependencies-even-though-they-exist

[the remark this answers is not in the document]
Comment (dknupp) [reply README.md:20]: Thanks @ddl-alexpanin -- I distinctly recall testing this in a previous PR, and I thought it had gone live:
https://test.pypi.org/project/python-domino-test/

I guess we never pushed to PyPI?

[the remark this answers is not in the document]
Comment (ddl-alexpanin) [reply README.md:20]: @dknupp yep, never. @ddl-gabrielhaim found https://pypi.org/project/domino-python/ which is not ours! Probably an extra argument against releasing as `python-domino` :)

**Review by dknupp — APPROVED** [review]

**Pull request merged**